community-membership.md

```diff
@@ -29,7 +29,7 @@ below.
 
 ## Member
 
-Members are continuously active contributors in the community.  They can have
+Members are *continuously active* contributors in the community. They can have
```

Comment:
Can we link the `Inactive Members` section towards the end of this para? I think going into the requirements, knowing what counts as inactive could help understand the process better.